src/PointwiseFunctions/Hydro/ComovingMagneticField.cpp

```diff
@@ -10,6 +10,39 @@
 
 namespace hydro {
 
+template <typename DataType>
+void comoving_magnetic_field(
+    const gsl::not_null<tnsr::A<DataType, 3>*> result,
+    const tnsr::I<DataType, 3>& spatial_velocity,
+    const tnsr::I<DataType, 3>& magnetic_field,
+    const Scalar<DataType>& magnetic_field_dot_spatial_velocity,
+    const Scalar<DataType>& lorentz_factor, const tnsr::I<DataType, 3>& shift,
+    const Scalar<DataType>& lapse) {
+  get<0>(*result) = get(lorentz_factor) *
+                    get(magnetic_field_dot_spatial_velocity) / get(lapse);
+
+  for (size_t i = 0; i < 3; ++i) {
+    result->get(i + 1) =
+        (magnetic_field.get(i) / get(lorentz_factor)) +
+        get(lapse) * get<0>(*result) *
+            (spatial_velocity.get(i) - shift.get(i) / get(lapse));
```

Comment:
why not analytically multiply through the lapse? Then you save yourself a division, which is expensive.

Reply:
done.